app/controllers/amr_heatmap_controller.rb

```diff
@@ -0,0 +1,45 @@
+class AmrHeatmapController < ApplicationController
+  include ReportHelper
+
+  before_action :admin_required
+
+  current_power do # Put this here for CLI
```

Comment:
There are special CLI-related situations in certain controllers where we'll need this block. We don't need this here since it isn't CLI-related. (CLI is our IDseq command line tool that users can use to upload data to IDseq)

Reply:
Removed!